index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width" />
+  <title>Jacob Schatz Website</title>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet" />
+  <link href="assets/css/styles.css" rel="stylesheet" type="text/css" />
+
+</head>
+
+<body>
+  <header>
+    <div id="hero-image-container">
+      <img id="header-logo" src="images/handsome_man.jpg" alt="hero image" />

```

Comment:
I avoid `id` at all cost. https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4